slides/01_intro.qmd

```diff
@@ -204,3 +215,14 @@ knitr::include_graphics("resources/pkg_graph.png")
     -   easier to document
 
 # Question, Comments? 
+
+## License information
+
+- Creators (initial authors): 
+    - Daniel Sabanes, [github.com/danielinteractive](https://github.com/danielinteractive/) and [linkedin.com/in/danielsabanesbove](www.linkedin.com/in/danielsabanesbove/)
```

Comment:
```suggestion
    - Daniel Sabanes Bove, [github.com/danielinteractive](https://github.com/danielinteractive/) and [linkedin.com/in/danielsabanesbove](www.linkedin.com/in/danielsabanesbove/)
```
plus how about reducing the required space by using the GitHub and LinkedIn font icons as we do it on the intro slides instead of giving the full link written out? the source code will have the full url in any case